src/planning/plan/tests/planificator-test.ts

```diff
@@ -62,10 +62,11 @@ describe('remote planificator', () => {
   async function createArc(options, storageKey) {
     const {manifestString, manifestFilename} = options;
     const loader = new Loader();
+    const storageService = new StorageServiceImpl();
     const context = manifestString
-        ? await Manifest.parse(manifestString, {loader, fileName: '', memoryProvider})
-        : await Manifest.load(manifestFilename, loader, {memoryProvider});
-    const runtime = new Runtime({loader, context, memoryProvider});
+        ? await Manifest.parse(manifestString, {loader, fileName: '', memoryProvider, storageService})
```

Comment:
you should never require a storage service when parsing. Parsing is completely independent of connecting to stores.

Reply:
removed